docs/supported-libraries.md

```diff
@@ -0,0 +1,138 @@
+| [JBoss EAP](https://www.redhat.com/en/technologies/jboss-middleware/application-platform) | 7.1.x, 7.3.x                | OpenJDK 8, 11    | Ubuntu 18, Windows Server 2019 |
+| [Jetty](https://www.eclipse.org/jetty/)                                                   | 9.4.x, 10.0.x, 11.0.x       | OpenJDK 8, 11    | Ubuntu 20                      |
+| [Payara](https://www.payara.fish/)                                                        | 5.0.x, 5.1.x                | OpenJDK 8, 11    | Ubuntu 18, Windows Server 2019 |
+| [Tomcat](http://tomcat.apache.org/)                                                       | 7.0.x, 8.5.x, 9.0.x, 10.0.x | OpenJDK 8, 11    | Ubuntu 18                      |
+| [TomEE](https://tomee.apache.org/)                                                        | 7.x, 8.x                    | OpenJDK 8, 11    | Ubuntu 18                      |
+| [Weblogic](https://www.oracle.com/java/weblogic/)                                         | 12                          | Oracle JDK 8     | Oracle Linux 7, 8              |
+| [Weblogic](https://www.oracle.com/java/weblogic/)                                         | 14                          | Oracle JDK 8, 11 | Oracle Linux 7, 8              |
+| [Websphere Liberty Profile](https://www.ibm.com/cloud/websphere-liberty)                  | 20.0.0.12                   | OpenJDK 8, 11    | Ubuntu 18, Windows Server 2019 |
+| [WildFly](https://www.wildfly.org/)                                                       | 13.0.x                      | OpenJDK 8        | Ubuntu 18, Windows Server 2019 |
+| [WildFly](https://www.wildfly.org/)                                                       | 17.0.1, 21.0.0              | OpenJDK 8, 11    | Ubuntu 18, Windows Server 2019 |
+
+## Disabled instrumentations
+
+Some instrumentations can produce too many spans and make traces very noisy.
+For this reason, the following instrumentations are disabled by default:
+
+- `jdbc-datasource` which creates spans whenever the `java.sql.DataSource#getConnection` method is called.
+
+To enable them, add the `otel.instrumentation.<name>.enabled` system property:
+`-Dotel.instrumentation.jdbc-datasource.enabled=true`
+
+### Grizzly instrumentation
+
+When you use
+[Grizzly](https://javaee.github.io/grizzly/httpserverframework.html) for
+Servlet-based applications, you get better experience from Servlet-specific
+support. As these two instrumentations conflict with each other, more generic
+instrumentation for Grizzly HTTP server is disabled by default. If needed,
+you can enable it by adding the following system property:
+`-Dotel.instrumentation.grizzly.enabled=true`
+
+## Suppressing specific auto-instrumentation
+
+See [suppressing specific auto-instrumentation](suppressing-instrumentation.md)
+
+## Logger MDC (Mapped Diagnostic Context) auto-instrumentation
+
+See [Logger MDC auto-instrumentation](logger-mdc-instrumentation.md)
```

Comment:
I think "suppressing instrumentation" may be better in agent-config.md

And Logger MDC might deserve it's own section on README, since it's different from everything else, as it's about configuring your _application_ instead of configuring the _agent_.

Reply:
Yup, love both of these.  Will change now.